tiledb/CMakeLists.txt

```diff
@@ -502,7 +502,11 @@ if (NOT WIN32)
 endif()
 
 # S3 dependencies
-if (TILEDB_S3)
+if (TILEDB_S3 AND TILEDB_VCPKG)
+  message(STATUS "The TileDB library is compiled with S3 support.")
+  find_package(AWSSDK_EP REQUIRED COMPONENTS s3)
```

Comment:
How about we entirely skip the `_EP` mechanism when using vcpkg?

Reply:
(or not, I guess it makes sense for dependencies that are not exclusively distributed via vcpkg)